app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -136,15 +136,14 @@ function trackCustomImpressions() {
     }
 
     // display add
-    var rotatingDisplayUnits = document.querySelectorAll('[data-rotating-display-unit]');
+    var displayAds = document.querySelectorAll('[data-display-unit]');
     if (rotatingDisplayUnits.length > 0
       && tokenMeta
       && !isBot
       && windowBigEnough
       && checkUserLoggedIn()) {
         var csrfToken = tokenMeta.getAttribute('content');
-        [].forEach.call(rotatingDisplayUnits, function(unit) {
-          // do whatever
+        [].forEach.call(displayAds, function(unit) {
```

Comment:
Do we still need this hack? Modern browsers should have support for `forEach` on `NodeList` and we don't really target old browsers.